clients/hadoopfs/src/main/java/io/lakefs/auth/AWSLakeFSTokenProvider.java

```diff
@@ -108,15 +107,39 @@ public Request<GeneratePresignGetCallerIdentityRequest> newPresignedRequest() th
 
     public String newPresignedGetCallerIdentityToken() throws Exception {
         Request<GeneratePresignGetCallerIdentityRequest> signedRequest = this.newPresignedRequest();
+        Map<String, ?> rawQueryParams = signedRequest.getParameters();
+        Map<String, String> params = new HashMap<>();
+        // check if the value is an array and join it with commas depends on the AWS SDK version
```

Comment:
Not sure what "depends on the AWS SDK version" means here.

Reply:
deleted